src/app/crash/session.cpp

```diff
@@ -127,8 +127,12 @@ const Session::Backups& Session::backups()
 bool Session::isRunning()
 {
   loadPid();
+  std::string processName = "aseprite";
+  #if LAF_WINDOWS
+    processName = processName + ".exe";
+  #endif
   if (m_pid)
-    return base::is_process_running(m_pid);
+    return base::is_process_running(m_pid, processName.c_str());
```

Comment:
I saw the changes in laf for supporting this and they look good. I'm just wondering if we should get the current process name instead of using a hardcoded string?

Reply:
Yes! I agree with you.